docs/get-started.md

```diff
@@ -20,7 +20,7 @@ For more details on how to create repositories using template, read the article
 
 ## After creating your repository
 
-The following steps apply to your new repository, not the upptime/upptime template repository.
+The following steps apply to your new repository, not the `upptime/upptime` template repository.
```

Comment:
I saw the repo name was formatted as code elsewhere so I did it here as well to keep consistent.